jupyterhub_singleuser_profiles/profiles.py

```diff
@@ -50,6 +51,7 @@ def gpu_mode(self, value):
       self._gpu_mode = None
   
   def load_profiles(self, secret_name="jupyter-singleuser-profiles", filename=None, key_name="jupyterhub-singleuser-profiles.yaml", username=None):
+    self.gpus = []
```

Comment:
I am mildly worried about the name - I first thought this is for "number of GPUs" and was confused a bit until I realized it is the configuration for GPUs. I know we have other sections named just `sized`  and `profiles`, but I would probably consider naming this one a bit more explicitely - `gpuConfig`, `gpuProfiles` or something along those lines